modules/troubleshooting-cluster-upgrader.adoc

```diff
@@ -0,0 +1,74 @@
+// Module included in the following assemblies:
+//
+// * upgrade/upgrade-roxctl.adoc
+:_mod-docs-content-type: PROCEDURE
+[id="troubleshooting_upgrader_{context}"]
+= Troubleshooting Cluster Upgrader
+
+[role="_abstract"]
+This section describes potential problems and their solutions when using legacy installation method for the secured cluster and enabling the automated updates.
+The errors described here can be found in the clusters view when the upgrader fails.
+
+== Upgrader is missing permissions
+
+.Symptom
+
+The following error is displayed in the cluster page:
+
+[quote]
+____
+Upgrader failed to execute PreflightStage of the roll-forward workflow: executing stage "Run preflight checks": preflight check "Kubernetes authorization" reported errors. This usually means that access is denied. Have you configured this Secured Cluster for automatically receiving upgrades?"
+____
+
+.Procedure
+
+* Ensure that the bundle for Secured Cluster was generated with future upgrades enabled: "_Cluster is configured to allow future automatic upgrades_" before clicking on "_Download YAML file and keys_".
+* If possible, remove that secured cluster and generate a new bundle making sure that future upgrades are enabled.
+* If recreation of the cluster is not desired or possible, the following steps can be taken alternatively:
+** Ensure that Service Account `sensor-upgrader` exists in the same namespace as Sensor.
+** Ensure that a ClusterRoleBinding exists (default name: `<namespace>:upgrade-sensors`) that grants the `cluster-admin` ClusterRole to the `sensor-upgrader` Service Account.
+
+== Upgrader cannot start due to missing image
+
+.Symptom
+
+The following error is displayed in the cluster page:
+
+[quote]
+____
+"Upgrade initialization error: The upgrader pods have trouble pulling the new image: Error pulling image: (...) (<image_reference:tag>: not found)"
+____
+
+.Procedure
+
+* Ensure that the Secured Cluster can access the registry and pull the image `<image_reference:tag>`.
+* Ensure that the image pull secrets are configured correctly in the Secured Cluster.
+
+== Upgrader cannot start due to an unknown reason
+
+.Symptom
+
+The following error is displayed in the cluster page:
+
+[quote]
```

Comment:
change formatting

Reply:
Done in 35b51ebb9727858bfb6e58580d0280d562444054